.github/workflows/github_actions.yml

```diff
@@ -0,0 +1,48 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+jobs:
+  build:
+    name: Creating Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: v ${{ github.ref }}
+          draft: false
+          prerelease: false
+  build:
```

Comment:
job name is the same as previous?

Reply:
Also, can't this be the same job, just different steps?

Reply:
Changed it to different job name. GH was throwing some errors with runs and uses so kept different job